DragaliaAPI.Database/Repositories/IInventoryRepository.cs

```diff
@@ -11,4 +11,7 @@ public interface IInventoryRepository : IBaseRepository
     DbPlayerMaterial AddMaterial(string deviceAccountId, Materials type);
     Task<DbPlayerMaterial?> GetMaterial(string deviceAccountId, Materials materialId);
     IQueryable<DbPlayerMaterial> GetMaterials(string deviceAccountId);
+    DbPlayerDragonGift AddDragonGift(string deviceAccountId, DragonGifts type);
+    Task<DbPlayerDragonGift?> GetDragonGift(string deviceAccountId, DragonGifts materialId);
```

Comment:
Why is the gifts method an IQueryable but the singular Gift method a Task?

IMO it is preferable to return an IQueryable so that the caller can customize the query before executing it. But whichever way you decide to do it they should be consistent